### PR TITLE
Mizanul login testing

### PR DESCRIPTION
**Pull request by MizanulH489**
hey, I updated the parseinfo.py file so with nested for loops, it can print out the classes and graduation requirements from the JSON and the output. It should work now. Let me know if it works for you

### CODE REVIEW

**Review by MMazenA — CHANGES_REQUESTED** [review]
looks good to me but it seems like you pulled course history and not course requirements. can you have it save to the output file and rename the python file to parse_info.py

**Comment by MizanulH489** [comment]
Updated parseinfo fileand changed the name

File: parse_info.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import json
+
+#Open jsonfile
+open_f=open('data.json')
```
Comment (MMazenA) [inline parse_info.py:4]: this needs to be saving into the data folder not in the current directory

**Review by MMazenA — CHANGES_REQUESTED** [review]

File: parse_info.py (hunk 1 above)
Comment (MizanulH489) [reply parse_info.py:4]: So to be sure it would save to the jsondata file right?

Comment (MizanulH489) [reply parse_info.py:4]: image
I'm guessing something like this?

**Comment by MizanulH489** [comment]
Changed the file name, and fixed to where it should be saved.

**Pull request merged**